src/backend/distributed/metadata/dependency.c

```diff
@@ -160,6 +161,54 @@ static ViewDependencyNode * BuildViewDependencyGraph(Oid relationId, HTAB *nodeM
 static Oid GetDependingView(Form_pg_depend pg_depend);
 
 
+PG_FUNCTION_INFO_V1(citus_get_dependencies_for_object);
+
+
+/*
+ * citus_unmark_object_distributed(classid oid, objid oid, objsubid int)
+ *
+ * removes the entry for an object address from pg_dist_object. Only removes the entry if
+ * the object does not exist anymore.
+ */
+Datum
+citus_get_dependencies_for_object(PG_FUNCTION_ARGS)
```

Comment:
should we define this somewhere in src/backend/distributed/test ?

Reply:
I think this would depend on the expected use.

@onderkalaci  do you see value in having this available on production systems? If only for development/testing I would indeed suggest to move the function to the test specific code.